apps/judicial-system/backend/src/app/modules/case/guards/rolesRules.ts

```diff
@@ -216,7 +217,7 @@ export const defenderTransitionRule: RolesRule = {
   role: UserRole.DEFENDER,
   type: RulesType.FIELD_VALUES,
   dtoField: 'transition',
-  dtoFieldValues: [CaseTransition.APPEAL],
+  dtoFieldValues: [CaseTransition.APPEAL, CaseTransition.WITHDRAW_APPEAL],
```

Comment:
Please note that the defender should only be able to withdraw an appeal if the case was appealed by the defence. This needs to be reflected in the `canActivate` method below.